README.md

```diff
@@ -0,0 +1,99 @@
+# Library
+
+Physical calculation package for the mechanics and geometry of overhead power lines.
+
+## Set up development environment
+
+You need python 3.11. You may have to install it manually (e.g. with pyenv).
+
+Then you may create a virtualenv, install dependencies and activate the env:
+
+    # create virtual env (if needed) and install dependencies (including dev dependencies)
+    poetry install
+    poetry shell  # activate virtual env
+
+Tip: if using VSCode/VSCodium, configure it to use your virtual env's interpreter.
+
+## How to format or lint code
+
+Once dev dependencies are installed, you may format and lint python files like this:
+
+    poetry run poe format
+    poetry run poe lint
+
+Use following command if you only want to check if files are correctly formatted:
+
+    poetry run poe check-format
+
+You may automatically fix some linting errors:
+
+    poetry run poe lint-fix
+
+Tip: if using VSCode/VSCodium, you may also use Ruff extension.
+
+## How to check typing
+
+In order to check type hints consistency, you may run:
+
+    poetry run poe typing
+
+## How to test
+
+### On the command line:
+
+    poetry run poe test
+
+### In VSCode:
+
+Configure VSCode to use your virtual env's interpreter.
+Open the Testing tab and configure tests using pytest.
+Click to run tests.
+
+## All in one
+
+You may run every check mentioned above with just one command:
+
+    poetry run poe checks
+
+## Exporting the library
+
+In order to build the library (wheel and tar.gz archive):
+
+    poetry build
+
+## How to serve the documentation
+
+```{bash}
+cd ./docs
+mkdocs serve -a localhost:8001
+```
```

Comment:
Here I suggest using the new poetry command `poetry run poe doc` instead of bash lines :)